BasicSample/Assets/Shared/Scripts/InitializeMixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    [InitializeOnLoad]
+    public class InitializeMixedRealityProjectSelectionWindow
```

Comment:
Will this run every time the editor is opened? Would that be the behavior we want?

I expected this would be a one-time setup, then once someone applied one of these settings, it wouldn't show automatically. There are definitely pros and cons to each option; I'm curious what your intent here is.

Reply:
This will open only when you open unity editor for the first time Keaton. If you want to use this next time there is a menu item on the top that can be used to bring this again as shown below:
![image](https://user-images.githubusercontent.com/69484678/162277689-f9e8d775-514e-49b8-bdb7-df07ac0f14f0.png)